spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteDataFilesSparkAction.java

```diff
@@ -172,14 +172,16 @@ public RewriteDataFiles.Result execute() {
         partialProgressEnabled
             ? doExecuteWithPartialProgress(plan, commitManager(startingSnapshotId))
             : doExecute(plan, commitManager(startingSnapshotId));
+    ImmutableRewriteDataFiles.Result result = resultBuilder.build();
 
     if (removeDanglingDeletes) {
       RemoveDanglingDeletesSparkAction action =
           new RemoveDanglingDeletesSparkAction(spark(), table);
       int removedCount = Iterables.size(action.execute().removedDeleteFiles());
-      resultBuilder.removedDeleteFilesCount(removedCount);
+      return result.withRemovedDeleteFilesCount(result.removedDeleteFilesCount() + removedCount);
```

Comment:
Making sure I follow the math so `result.removedDeleteFilesCount()` is how many orphan DVs are removed and the existing `removedCount` is how many dangling deletes are removed?

Reply:
yes correct

Reply:
can we rename the variables to make them more informative?
```
removedCount -> removedDangling
```

also add comment to explain `result.removedDeleteFilesCount()` captures the number of removed orphaned DVs

Reply:
`removedCount` includes dangling deletes of any kind. I've updated the variable name to `removedDeleteFiles`